### ISSUE TITLE
Better way for handling due date

### ISSUE THREAD

**Issue by wuethrich44** [nextcloud/tasks#9]
I want to move away from Google Inbox to Nextcloud Tasks, but I miss one handy feature. In Google Inbox you can change the due date for a task very easy with one click (see screenshot). In Nextcloud Task I always have to enter a valid date which is a little bit cumbersome. 

I can provide a PR with this feature if there is any interest.

@raimund-schluessler What do you think about that? (Sorry screenshot is in german but I guess you should understand it 😉)

![screenshot](https://cloud.githubusercontent.com/assets/6050639/20037460/a28f8a9c-a41f-11e6-9f41-9e8533490947.PNG)

**Comment by raimund-schluessler** [comment]
So, you basically propose to provide a couple of reasonable suggestions for the date and only manually enter a date if none of these suggestions fit?

I guess this could really improve the usability. @jancborchardt What do you think about this?

**Comment by wuethrich44** [comment]
Exactly so you can postbone a task just with one click. Outlook has a similar concept:

![screenshot](https://cloud.githubusercontent.com/assets/6050639/20038135/8efb289c-a42e-11e6-9710-c263eeca5c3a.PNG)

**Comment by jancborchardt** [comment]
Yeah @wuethrich44, that sounds awesome! :) If you do want to contribute with a PR that would be very cool! :tada:

**Comment by wuethrich44** [comment]
Ok great I will check your guidelines and provide a PR.

**Comment by mahibi** [comment]
any news on this?

**Comment by wuethrich44** [comment]
@mahibi Sorry I'm currently a little bit busy so I close this issue until I'll find some time to finish it.

**Comment by raimund-schluessler** [comment]
No need to close this, this is still an issue.

**Comment by armaccloud** [comment]
Could one of the options be the exclusion of time / only choosing a date?

Especially for tasks, often you just want to have them done on a certain day or several days, not specific times. It takes quite some effort choosing a time for each task.

**Comment by raimund-schluessler** [comment]
> Could one of the options be the exclusion of time / only choosing a date?

You can already set the task to all-day. Then you don't have to put a time, only a date for start and end

**Comment by mikezieg** [comment]
any news on this?

**Comment by jakobroehrl** [comment]
Maybe I could implement it like this?
https://github.com/nextcloud/deck/pull/2710
